sdk/containerregistry/azure-containerregistry/samples/sample_delete_old_tags.py

```diff
@@ -42,15 +38,15 @@ def delete_old_tags(self):
         client = ContainerRegistryClient(account_url, credential)
 
         for repository in client.list_repository_names():
-            repository_client = ContainerRepositoryClient(account_url, repository, credential)
+            print(repository)
             # [END list_repository_names]
 
             # [START list_tags]
             tag_count = 0
-            for tag in repository_client.list_tags(order_by=TagOrder.LAST_UPDATE_TIME_DESCENDING):
+            for tag in client.list_tags(repository, order_by=TagOrder.LAST_UPDATE_TIME_DESCENDING):
                 tag_count += 1
                 if tag_count > 3:
-                    repository_client.delete_tag(tag.name)
+                    client.delete_tag(repository, tag.name)
```

Comment:
hmm so we want delete all tags after the first 3? Maybe this would work?
```python
tags = client.list_tags(...)
for tag in list(tags)[3:]:
    client.delete_tag(...)
```